core-library/src/main/java/org/silverpeas/core/contribution/content/form/displayers/WysiwygFCKFieldDisplayer.java

```diff
@@ -659,17 +504,13 @@ public void cloneContents(ResourceReference fromPK, ResourceReference toPK, Map<
   }
 
   public void mergeContents(String componentIdFrom, String objectIdFrom, String componentIdTo,
-      String objectIdTo) throws UtilException, IOException {
+      String objectIdTo) throws IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)